overlay.nix

```diff
@@ -13,12 +13,40 @@ _finalNixpkgs: prevNixpkgs: let
   circ = prevNixpkgs.callPackage ./packages/circ/default.nix {};
 
   wasmd = prevNixpkgs.callPackage ./packages/wasmd/default.nix {};
+
+  cattrs22-2 = prevNixpkgs.python3Packages.cattrs.overrideAttrs (finalAttrs: previousAttrs: {
```

Comment:
Can you leave a comment here explaining why this specific version is necessary?

Reply:
Comment added.